app/views/govuk_wrapper.scala.html

```diff
@@ -72,13 +72,6 @@
 
 @mainContentHeader = {
 
-    @if(appConfig.languageTranslationEnabled) {
-        @views.html.language_selection(
-            appConfig.languageMap,
-            appConfig.routeToSwitchLanguage,
-            Some("translate u-align--right"))
-    }
-
```

Comment:
Is it not a GDS requirement to have translation enabled?

Reply:
I asked and have been told to remove this and been told this service will not be translated.

Reply:
👍